### DISCUSSION TITLE
eTag is not showing in Response Header

### DISCUSSION THREAD

**Discussion by tornado1512** [Adobe-Consulting-Services/acs-aem-commons#3336] [General]
## Required Information

1. AEM Version: 6.5.15.
2. ACS AEM Commons Version:4.6.0.

### Expected Behavior

We have created an OSGi configuration and did other configurations for enabling eTag in response header.
Post that in response header eTag key value pair should visible.
As per the document https://adobe-consulting-services.github.io/acs-aem-commons/features/etag/index.html 

### Actual Behavior

After doing these configuration we are still not able to see the eTag in response header.

### Steps to Reproduce

I have followed this document: 
https://adobe-consulting-services.github.io/acs-aem-commons/features/etag/index.html 

### Links
https://adobe-consulting-services.github.io/acs-aem-commons/features/etag/index.html

**Comment by kwin** [comment]
Can you check directly on AEM (not via Dispatcher/Webserver)? Usually the issue is that the webserver removes/replaces those headers without config changes.
